src/i18ndude/generator.py

```diff
@@ -10,6 +10,8 @@
 
 import re
 
+NOT_ALLOWED_IN_NAME = re.compile('[^\w-]')
```

Comment:
I was going to say "please include underscore in here", but apparently underscore is included in the `\w`.

Please make it a raw string though (`r'[^\w-]'`), otherwise on Python 3 you get: `DeprecationWarning: invalid escape sequence \w`. (I see this is true for a few existing regular expressions too, which I should fix, but let's not introduce others.)